tests/modules/nf-core/samtools/calmd/main.nf

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SAMTOOLS_CALMD } from '../../../../../modules/nf-core/samtools/calmd/main.nf'
+
+workflow test_samtools_calmd {
+    
+    input1 = [
+        [ id:'test', single_end:false ], // meta map
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/genomics/homo_sapiens/illumina/bam/test.paired_end.sorted.bam", checkIfExists: true)
+    ]
+
+    input2 = file("https://github.com/nf-core/test-datasets/raw/modules/data/genomics/homo_sapiens/genome/genome.fasta", checkIfExists: true)
+    SAMTOOLS_CALMD ( input1, input2 )
+}
```

Comment:
Does this accept CRAM as well? If so, can we add a test for it?

Reply:
It does, but apparently pointless:

```
Calmd can also read and write CRAM files although in most cases it is pointless as CRAM recalculates MD and NM tags on the fly. The one exception to this case is where both input and output CRAM files have been / are being created with the no_ref option.
```

Reply:
http://www.htslib.org/doc/samtools-calmd.html

Reply:
It does. However, these tags are generated automatically when CRAMs are produced so there are no datasets in test-datasets that will work. I have generated a test dataset using one of the bams on there and not specifying a reference but I think I now need to make a pull request to add that dataset?